Topic 3/src/main/java/bootcamp/exercises/ConvertToText.java

```diff
@@ -0,0 +1,128 @@
+package bootcamp.exercises;
+
+public class ConvertToText {
+	 private final String[] UNITS = {"", "one", "two", "three", "four","five","six","seven","eight","nine"};
+
+	  private final String[] TENS = {"ten","eleven","twelve","thirteen","fourteen","fifteen","sixteen","seventeen","eighteen","nineteen", "twenty", "thirty","forty","fifty","sixty","seventy","eighty","ninety"};
+
+		  
+	  public String convert(String number) {
+		  String intPart= "";
+		  String decimalPart;
+		  number= number.replace(".", ",");
+		  if(number.indexOf(",")==-1){
+			  number= number+ ",00";
+		  }
+		  String Num[]= number.split(",");
+		  decimalPart= Num[1];
+		  int lenght= Num[0].length();
+		  int lenght2= decimalPart.length();
+		  if((lenght< 10) && (lenght2 <3)){
+			  decimalPart="and " + decimalPart + "/100 dollars.";
+			  String integerPart= Num[0];
+			  if(Integer.parseInt(integerPart) == 0){
+				  intPart= "cero ";
+			  }
+			  else {
+				  if(Integer.parseInt(integerPart) > 999999){
```

Comment:
[Medium] Instead of having nested if, you can return the number when you found the proper method to treat it.
For example:
```java
if (condition a) {
  return value;
}
if (condition b) {
  return value;
}
```

or

```java
if (condition a) {
  statement 1;
} else if (condition b) {
  statement 2;
} else if (condition c) {
  statement 3;
} else {
  statement 4;
}
return value;
```